packages/mask/src/plugins/Tips/SNSAdaptor/components/TipsRealmContent/index.tsx

```diff
@@ -89,11 +93,24 @@ export const TipsRealmContent: Plugin.InjectUI<Plugin.SNSAdaptor.TipsRealmOption
 
     if (slot === Plugin.SNSAdaptor.TipsSlot.MirrorMenu) {
         return (
-            <Guide>
-                <Stack display="inline-block" width="38px" height="38px" position="relative" top={2}>
-                    {button}
-                </Stack>
-            </Guide>
+            <PluginGuideProvider
+                value={{
+                    storageKey: EnhanceableSite.Mirror,
+                    totalStep: 1,
```

Comment:
You can compute it from `guides.length`.

Reply:
The `totalStep` is clear and understandable for developer.  I think should keep it.

Reply:
No need to let the user of `<PluginGuideProvider />` to concern about it.